<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaDelete</class>
 <widget class="QWidget" name="telaDelete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deletar</string>
  </property>
  <widget class="QLineEdit" name="inputID">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-style: outset;
border-width: 1px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(205, 217, 255);
border-style: outset;
border: 2px solid black;</string>
   </property>
   <property name="text">
    <string>DELETAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVoltarTelaMainD">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(205, 217, 255);
border-radius: 10px;
border-style: outset;
border: 2px solid black;</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
